bindata/network/ovn-kubernetes/ovnkube-node.yaml

```diff
@@ -293,6 +293,9 @@ spec:
           name: ovn-cert
         - mountPath: /ovn-ca
           name: ovn-ca
+        - mountPath: /lib/modules
```

Comment:
This commit (`Allow ovnkube-node to load kernel modules `) should not be needed any more; kubelet will always have created iptables rules before any pods run, so now that we've updated kubelet to realize it should be doing IPv6 iptables rules, ovn-kubernetes won't need to load the module itself.

Reply:
OK - we can remove this and find out pretty quickly if it works!